pkg/api/api.go

```diff
@@ -391,6 +391,34 @@ func (s *server) newTracingHandler(spanName string) func(h http.Handler) http.Ha
 	}
 }
 
+func (s *server) contentLengthMetricMiddleware() func(h http.Handler) http.Handler {
+	return func(h http.Handler) http.Handler {
+		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			now := time.Now()
+			h.ServeHTTP(w, r)
+			switch r.Method {
+			case http.MethodGet:
+				{
```

Comment:
the curly braces on this level of nesting are not needed

Reply:
i added for visual clarity, can remove it